### DISCUSSION TITLE
Question about merging clusters

### DISCUSSION THREAD

**Discussion by mkrzywonski** [nextcloud/recognize#388] [General]
My photo library has photos of individuals from when they were babies to their full grown adulthood. I'm curious about how merging these clusters of the same person will affect future recognition. If I name faces of my baby pictures with my name, then later merge adult pictures of myself into that same person, do future recognition attempts of newly added photos use the average of all those faces to identify me? Or are the baby pictures the standard for identifying me because I named them first? Or maybe it doesn't matter and pictures of me both as a baby and as an adult should be recognized just fine in the future?  

Also, If I have named people in my photos will that information also be applied to photos belonging to other users? So, will I already be identified in my wife's photos because I added my name to my recognized photos in my own albums?

Thanks.

**Comment by marcelklehr** [comment]
Thank you for the great question! New photos currently always trigger a new clustering internally. If your new photos appear in a cluster with photos that exclusively belong to person A, then these new photos will also be added to person A. Otherwise, recognize will create a new person for these new photos. Since your baby photos and your adult photos will likely be in two internal clusters, once you merge them into one person, additional photos should be classified correctly.

Persons as well as names are stored per user account. You cannot share people across user accounts currently.
